code/modules/hydroponics/grown/cannabis.dm

```diff
@@ -2,7 +2,6 @@
 /obj/item/seeds/cannabis
 	name = "pack of cannabis seeds"
 	desc = "Taxable."
-	icon = 'goon/icons/obj/hydroponics.dmi'
```

Comment:
I'm pretty sure he mentioned in IRC earlier something about moving the icons to the main file.
which is a no-no for goon icons.